core/vm/evm.go

```diff
@@ -448,6 +449,12 @@ func openDB() {
 		PRIMARY KEY (address, chain),
 		FOREIGN KEY (creationCodeID) REFERENCES creationCode(id),
 		FOREIGN KEY (codeHashID) REFERENCES codeHash(id));`,
+		`CREATE TABLE IF NOT EXISTS complete (
+			address CHAR(42) NOT NULL,
+			chain VARCHAR(64) NOT NULL,
+			code bytea NOT NULL,
+			hash CHAR(66) NOT NULL,
```

Comment:
Does it make sense to store this as raw bytes instead of hex encoded? The same for address.

Reply:
Are you asking about `address` and `hash` columns?

Reply:
Yes. Also can `chain` be an integer?

Reply:
Ligi proposed to use this for chainID https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md so that is why it is not integer

Reply:
ok!